rpcserver.go

```diff
@@ -991,10 +992,14 @@ func (s *rpcServer) parseRPCOutputs(outputs []*poolrpc.Output) ([]*wire.TxOut,
 		if err != nil {
 			return nil, err
 		}
-		res = append(res, &wire.TxOut{
+		txOut := &wire.TxOut{
 			Value:    int64(output.ValueSat),
 			PkScript: outputScript,
-		})
+		}
+		if txrules.IsDustOutput(txOut, txrules.DefaultRelayFeePerKb) {
```

Comment:
Perhaps we should move this check further into the pipeline (within the account manager) so we can bolster things with some additional unit tests?